LondonBritishLibrary/orient/BLorient13266.xml

```diff
@@ -0,0 +1,419 @@
+                                    </item>
+                                </list>
+                            </additions>  
+                        </physDesc>
+                        <history>
+                            <origin>
+                                <origDate notBefore="1700" notAfter="1800">18th century</origDate>
+                            </origin>
+                        </history>
+                    </msPart>
+                    <msPart xml:id="p2">
+                        <msIdentifier>
+                            <idno/>
+                        </msIdentifier>
+                        <msContents>
+                            <msItem xml:id="p2_i1">
+                                <locus from="194ra" to="201vb"/>
+                                <title ref="LIT1356eqaban"/>
+                                <incipit xml:lang="gez">ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ ዕቀበኒ፡ በእንተ፡ ዘኀደርከ፡ በከርሠ፡ ማርያም። ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ ዕቀበኒ፡ በእንተ፡ ማርያም፡
+                                    አፍቂረከ፡ ዘኀደርከ፡ በውስተ፡ ማኅፀና፡ ለማርያም፡ ድንግል። ዕቀበኒ፡ <corr resp="PRS8999Strelcyn">ኦ</corr>እግዚእየ፡ በእንተ፡ ፍቅሩ፡ ወበእንተ፡
+                                    ዘሐቀፍኪዮ፡ በአእዳውኪ፡ ወበእንተ፡ ዘአጥበውኪዮ፡ ሐሊበ፡ ውእቱ፡ እግዚእነ፡ እየሱስ፡ ክርስቶስ። ዕቀበኒ፡ በእንተ፡ ፍቅራ፡ ለማርያም፡ እግዚአየ፡ ኢየሱስ፡ ክርስቶስ፡ 
+                                        </incipit>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">6</measure>
+                                        <dimensions type="outer" unit="mm">
+                                            <height>215</height>
+                                            <width>180</width>
+                                        </dimensions>
+                                    </extent></supportDesc>
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="20"><locus from="196" to="201"/></layout>
+                                </layoutDesc>
+                            </objectDesc> 
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h2">
+                                    <desc>Careful handwriting</desc>
+                                    <date notBefore="1700" notAfter="1800" resp="PRS8999Strelcyn"/>
+                                </handNote>
+                            </handDesc>
+                            <additions>
+                                <list>
+                                    <item xml:id="e4">
```

Comment:
does the catalogue state that e4 and e5 are additions? I think these comments are a normal part of the work